README.md

```diff
@@ -175,6 +175,14 @@ It is important to note as well that if a user is binned into the Default compon
 
 This project is written using ES6 and all build steps / transpilation are done by webpack. In order to develop and test locally, it is necessary to simply run the ```build.sh``` shell script which will take care of transpiling to ES5.
 
+If you have not already, you may need to install babel and webpack:
```

Comment:
are these not installed when you run ```npm install```? If not, we should add them to the devDependencies in the package.json file.